Source: @pullmoll
Upstream: no
Reason: Use the proper interface of libressl which does not expose the internal structure of SSL

--- a/libsrc/Wi/http.c	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/http.c	2020-01-27 15:09:53.284905265 +0100
@@ -9949,7 +9949,7 @@
 	  cli_ssl_get_error_string (err_buf, sizeof (err_buf));
 	  sqlr_new_error ("42000", "..002", "SSL_do_handshake failed %s", err_buf);
 	}
-      ssl->state = SSL_ST_ACCEPT;
+      SSL_set_accept_state (ssl);
       while (SSL_renegotiate_pending (ssl) && ctr < 1000)
 	{
 	  timeout_t to = { 0, 1000 };
